plugins/gpio/GPIODriver.cpp

```diff
@@ -144,9 +153,19 @@ bool GPIODriver::SetupGPIO() {
       failed = true;
       break;
     }
-    if (write(fd, direction.c_str(), direction.size()) < 0) {
-      OLA_WARN << "Failed to enable output on " << str.str() << " : "
-               << strerror(errno);
+    // Assign correct initial state
+    if (inverted) {
+        direction = &inverted_direction;
+    } else {
+        direction = &normal_direction;
+    }
+
+    OLA_DEBUG << "Configuring GPIO pin " << static_cast<int>(*iter)
+              << " with " << (inverted ? "inverted" : "normal") << " logic";
+    if (write(fd, direction->c_str(), direction->size()) < 0) {
+      OLA_WARN << "Failed to enable output on " << str.str()
+               << " with " << (inverted ? "inverted" : "normal") << " logic"
```

Comment:
Given this is showing the full path to the pin, perhaps also show the direction string here, rather than the friendly inverted/normal output.

Or turn it on it's head and put that in the DEBUG now we've added it and just make the WARN more user friendly "Failed to enable output on pin n as high/low".